todoapp/app/build.gradle

```
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
+    buildToolsVersion '28.0.3'
```

We have bumped up the build tools version, so there is no need to hardcode the version here. May I suggest we use the `buildToolsVersion` variable so that we only need to update the version in one place when the need arises.

```suggestion
    buildToolsVersion rootProject.ext.buildToolsVersion
```